tests/python/gaia-ui-tests/gaiatest/apps/contacts/regions/contact_form.py

```diff
@@ -144,13 +144,16 @@ def __init__(self, marionette):
         update = Wait(self.marionette).until(expected.element_present(*self._update_locator))
         Wait(self.marionette).until(lambda m: update.location['y'] == 0 and update.is_displayed())
 
-    def tap_update(self, return_details=True):
+    def tap_update(self, return_details=True, return_class=None):
```

Comment:
My bad too, my command wasn't correct, it should have been: `git grep 'tap_update.return'`

One file will be affected: `gaiatest/tests/functional/messages/test_add_to_an_existing_contact_from_messages.py`